Yoast/wordpress-seo
# Update heading styling

## Summary
This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* For this PR, i set some global styles within the header.scss to not mess with @JoseeWouters PR's where she removes a bunch of code. This should be split out later on.

## Test instructions
This PR can be tested by following these steps:

* check if all headings across all admin pages look and feel as the design.
* makes ure the te post-overview and post-editor does not change.

## UI changes

## Documentation

## Quality assurance


Fixes #14289 & https://github.com/Yoast/wordpress-seo/issues/14286
